app/controllers/v0/terms_of_use_agreements_controller.rb

```diff
@@ -2,11 +2,18 @@
 
 module V0
   class TermsOfUseAgreementsController < ApplicationController
+    def latest
+      terms_of_use_agreement = get_terms_of_use_agreements_for_version(params[:version]).last
+      render_success(terms_of_use_agreement, :ok)
+    rescue => e
```

Comment:
Not a big deal, but the other controller actions are rescuing named exceptions whereas this one just rescues anything that can go wrong. Might be worthwhile writing a log entry with the exception details so we have some visibility into errors.

Reply:
Actually, I just decided to removing non-named exceptions in this controller. Any exceptions we don't expect should be a 500 and then sent to sentry